### ISSUE TITLE
Multi gpu, named tuple converts to tuple

### ISSUE THREAD

**Issue by Vozf** [PyTorchLightning/pytorch-lightning#6257]
## 🐛 Bug

When setting multiple gpu training, named tuples are converted to tuples and this breaks training resulting in error
```
    preds = self.model(batch.image)
AttributeError: 'tuple' object has no attribute 'image'
```
Where batch is a named tuple and works fine as namedtuple when training with single gpu

**Comment by Vozf** [comment]
This may be connected to https://github.com/PyTorchLightning/pytorch-lightning/issues/1588

**Comment by edenlightning** [comment]
Thanks for the report!
This should be resolved by PyTorch 1.8. This isn't an issue in Lightning.

A workaround in the meantime: use dictionaries instead.
